apps/client-asset-sg/src/styles.scss

```diff
@@ -133,12 +143,18 @@ input {
   position: relative;
 }
 
+.disclaimer-backdrop-color {
+  background-color: grey;
+}
```

Comment:
Move this back into the disclaimer CSS (using `::ng-deep`) - Its not a problem to have "global" css in the component modules, as long as it is clean (i.e. as safe as possible) and commented as such.